changelog.md

```diff
@@ -1,3 +1,107 @@
+* new inspector issues gatherer for Audit.IssueAdded events ([#10664](https://github.com/GoogleChrome/lighthouse/pull/10664))
+* renderer: fix null Util.i18n in PSI renderer ([#10822](https://github.com/GoogleChrome/lighthouse/pull/10822))
+* subRow refactor, rename to subItem ([#10867](https://github.com/GoogleChrome/lighthouse/pull/10867))
+
+## Experimental
+
+Features hidden behind the `--preset=experimental` flag.
+
+* legacy-javascript: reduce polyfills, fix core-js import in test ([#10937](https://github.com/GoogleChrome/lighthouse/pull/10937))
+* legacy-javascript: use prescriptive language in title ([#10850](https://github.com/GoogleChrome/lighthouse/pull/10850))
+* legacy-javascript: fix core-js 3 detection ([#10852](https://github.com/GoogleChrome/lighthouse/pull/10852))
+* legacy-javascript: use third-party-web for scoring ([#10849](https://github.com/GoogleChrome/lighthouse/pull/10849))
+* duplicated-javascript: display transfer size ([#10701](https://github.com/GoogleChrome/lighthouse/pull/10701))
+
+## Deps
+
+* axe-core: upgrade to 3.5.5 ([#10986](https://github.com/GoogleChrome/lighthouse/pull/10986))
+* upgrade chrome-launcher@0.13.3 ([#10911](https://github.com/GoogleChrome/lighthouse/pull/10911))
+* snyk: update snyk snapshot ([#10840](https://github.com/GoogleChrome/lighthouse/pull/10840), [#10940](https://github.com/GoogleChrome/lighthouse/pull/10940), [#10980](https://github.com/GoogleChrome/lighthouse/pull/10980), [#11010](https://github.com/GoogleChrome/lighthouse/pull/11010))
+* remove bundlesize ([#10999](https://github.com/GoogleChrome/lighthouse/pull/10999))
+* update to lhci 0.4.0 ([#10828](https://github.com/GoogleChrome/lighthouse/pull/10828))
+
+## Report
+
+* use KiB instead of KB ([#10870](https://github.com/GoogleChrome/lighthouse/pull/10870))
+* metrics: use css grid so metrics are aligned ([#10789](https://github.com/GoogleChrome/lighthouse/pull/10789))
+* don't dim disclaimer anchor links ([#10981](https://github.com/GoogleChrome/lighthouse/pull/10981))
+* use acronyms and round metrics for shorter calc url ([#10954](https://github.com/GoogleChrome/lighthouse/pull/10954))
+* update link for budgets audit ([#10944](https://github.com/GoogleChrome/lighthouse/pull/10944))
+* add trailing slash to web.dev links ([#10967](https://github.com/GoogleChrome/lighthouse/pull/10967))
+* fix the width of the 3-dots menu in topbar ([#10855](https://github.com/GoogleChrome/lighthouse/pull/10855))
+* fix zebra styling, sub item rows for 3p filter ([#10978](https://github.com/GoogleChrome/lighthouse/pull/10978))
+* updated method signature typing to remove focusevent cast ([#10858](https://github.com/GoogleChrome/lighthouse/pull/10858))
+
+## Clients
+
+* psi: show disclaimer and calclink ([#10936](https://github.com/GoogleChrome/lighthouse/pull/10936))
+
+## I18n
+
+* adjust LCP element description ([#11018](https://github.com/GoogleChrome/lighthouse/pull/11018))
```

Comment:
report??

Reply:
cc @exterkamp fyi I think we want to keep `i18n:` to just things that affect the i18n system, not that touch any string

Reply:
Makes sense. Been awhile since I've used these tags 😟

Reply:
I think that was actually my bad. I think it was `core(i18n):` and I was like "don't we have an i18n separate scope? and changed it before merge".

I kinda stand by the fact that this was an adjustment raised by i18n clarity purposes, but agreed generally strings shouldn't be `i18n` scope. Maybe `core(lcp): adjust element description` would've been best?

report feels weird since it has nothing to do with HTML report, it applies to all LHR uses

Reply:
> report feels weird since it has nothing to do with HTML report, it applies to all LHR uses

yeah, it feels a little weird but I usually put these in core